pylossless/rejection.py

```diff
@@ -0,0 +1,94 @@
+# Authors: Christian O'Reilly <christian.oreilly@sc.edu>
+#          Scott Huberty <seh33@uw.edu>
+#
+# License: MIT
+
+"""Class to implement a policy on how to apply flags to reject artifacts."""
+
+import numpy as np
+
+from .config import Config
+
+
+class RejectionPolicy:
+    """Class used to implement a rejection policy for a pipeline output."""
+
+    def __init__(self, config_fname=None):
+        """Initialize class.
+
+        Parameters
+        ----------
+        config_fname : pathlib.Path
+            path to config file specifying the parameters to be used
+            in for the rejection policy.
+        """
+        self.ch_flags_to_reject = ['ch_sd', 'low_r', 'bridged']
+        self.epoch_flags_to_reject = []
+        self.ic_flags_to_reject = ['muscle', 'heart', 'ch_noise', 'eye']
+
+        self.ic_rejection_threshold = 0.3
+
+        # Can be:
+        #   "": Do nothing aside from adding them to raw.info["bads"]
+        #   "drop": Drop bad channels.
+        #   "interpolate": Interpolate bad channels.
+        self.clean_ch_mode = ""
```

Comment:
can we change the default to `None`? That seems more idiomatic to me. 

Can we allow this to be set in the constructer? i.e. `clean_ch_mode=None` by default, and `self.clean_ch_mode=clean_ch_mode`